### ISSUE TITLE
Disable / Enable notification and unread-count for bundle

### ISSUE THREAD

**Issue by alayek** [Thomas101/wmail#658]
* Which version of WMail are you using? [e.g. 1.3.1]
> WMail version 2.3.0

* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]
> OSX 10.10 El Capitan

**Feature Request**
It would be nice to disable notification and unread count for certain bundles. Like, bundle for banking transaction alerts. 

I would like to get these updates directly in my inbox for future references, but I don't want them to increase unread count or raise a notification.

**Comment by Thomas101** [comment]
Definitely! There are some extra settings in 2.3.0 for disabling notifications from all bundles but adding it for specific ones would be a great feature

**Comment by alayek** [comment]
I could add a PR if you'd like 😃
